storage.bs

```diff
@@ -151,16 +151,26 @@ must be cleared in its entirety.
 <h2 id=persistence>Persistence permission</h2>
 
 A <a>box</a> can only be turned into a <a>persistent box</a> if the user (or user agent
-on behalf of the user) has granted <dfn>permission</dfn>.
+on behalf of the user) has granted permission to use the {{"persistent-storage"}} feature.
 
 <p class="note">When granted to an <a>origin</a>, the persistence permission can be used to protect
 storage from the user agent's clearing policies. The user agent cannot clear storage marked as
 persistent without involvement from the <a>origin</a> or user. This makes it particularly useful for
 resources the user needs to have available while offline or resources the user creates locally.
 
-<p class="XXX">We will eventually integrate with the Permissions API, with the identifier
-"<dfn export for=PermissionName type=enum-value><code>persistent-storage</code></dfn>", but since
-that specification is not in great shape at the moment that has not happened yet.
+The <dfn for="PermissionName" enum-value>"<code>persistent-storage</code>"</dfn> <a>powerful feature</a>'s
+permission-related flags, algorithms, and types are defaulted, except for:
+
+<dl>
+ <dt><a>permission state</a></dt>
+ <dd><code>{name: {{"persistent-storage"}}}</code>'s <a>permission state</a> must have the same
+ value for all <a>environment settings objects</a> with a given <a>origin</a>.</dd>
```

Comment:
I'm happy to start a thread, but I'm not really sure where to start.  It seems like we want a generalized model since some want to expose generalized "get" and "request" operations. It's not clear how those would work if there wasn't a generalized model.

And if there's no generalized store that means each permission either has to adopt a default store or implement its own. We could go down that route too, but I'm not sure that's more useful than just allowing the nature of the key to be changed on a per-permission-type basis.